### PR TITLE
Update Iran

### PR DESCRIPTION
**Pull request by fakhamatia**

### CODE REVIEW

**Review by rootmelo92118 — APPROVED** [review]

File: data/category-ir-scholar [hunk 1]
```diff
@@ -3,7 +3,6 @@
 ac.ir
 edu.ir
 iau.ac.ir
-iranlms.ir
```
Comment (rootmelo92118) [inline data/category-ir-scholar:6]: Why removed this domain?

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

File: data/category-ir-scholar (hunk 1 above)
Comment (fakhamatia) [reply data/category-ir-scholar]: > Why removed this domain?

I thought iranlms.ir was one with iranlms.org.

**Review by IceCodeNew — APPROVED** [review]

File: data/category-ir-scholar (hunk 1 above)
Comment (IceCodeNew) [reply data/category-ir-scholar]: @rootmelo92118 please update your opinion on this PR. Should we give it a go?

**Comment by IceCodeNew** [comment]
@fakhamatia please make another PR for changes that are not related to the title next time.

**Pull request merged**